tests/common.sh

```diff
@@ -225,12 +225,18 @@ function wait_for() {
 }
 
 function error {
+  if [[ $ERRORED == "true" ]]
+  then
+    exit
```

Comment:
so we get logs on first attempt alone if im reading it right

Reply:
Correct. one of the potential issues was if it tries to dump the logs and breaks. the trap would send it back to here and it would just go and go. This way we only try and dump the logs once and then bugger out.